content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -351,8 +352,15 @@ link:http://localhost:8080/pipeline-syntax/globals#env[localhost:8080/pipeline-s
 assuming a Jenkins master is running on `localhost:8080`, and includes:
 
 BUILD_ID:: The current build ID, identical to BUILD_NUMBER for builds created in Jenkins versions 1.597+
-JOB_NAME:: Name of the project of this build, such as "foo" or "foo/bar".
+BUILD_NUMBER:: The current build number, such as "153"
+BUILD_TAG:: String of jenkins-\$\{JOB_NAME}-\$\{BUILD_NUMBER}. Convenient to put into a resource file, a jar file, etc for easier identification
```

No need to escape both.  When both are escaped, the backslash renders on the page.
```suggestion
BUILD_TAG:: String of jenkins-$\{JOB_NAME}-$\{BUILD_NUMBER}. Convenient to put into a resource file, a jar file, etc for easier identification
```